src/edu/ucsb/cs56/projects/games/states_quiz/GamePanel.java

```diff
@@ -111,13 +119,32 @@ private JButton generateHintButton(int x, int y, int w, int h, String text) {
         hintButton.setVisible(false);
         hintButton.setBounds(x, y, w, h);
         hintButton.addActionListener(e -> {
+<<<<<<< HEAD
             State state = mapPanel.getQuestionManager().getCorrectState();
-            //hintButton.setText(this.getStateQuadrant(state.getXCoord(), state.getYCoord())+"\nThe first letter of the capital is "+ getFirstLetterOfCapital(state.getCapital()));
+            hintButton.setText(this.getStateQuadrant(state.getXCoord(), state.getYCoord())+"\nThe first letter of the capital is "+ getFirstLetterOfCapital(state.getCapital()));
             String stateHint = GamePanel.this.getStateQuadrant(state.getXCoord(), state.getYCoord());
             String capitalHint = "Capital's first letter: " + getFirstLetterOfCapital(state.getCapital());
             hintButton.setText("<html>State is " + stateHint + " " + capitalHint + "</html>");
         });
 
+=======
+		State state = mapPanel.getQuestionManager().getCorrectState();
+		//hintButton.setText(this.getStateQuadrant(state.getXCoord(), state.getYCoord())+"\nThe first letter of the capital is "+ getFirstLetterOfCapital(state.getCapital()));
+		String stateHint;
+		if (state.getName().equals("Alaska")) {
```

Comment:
And this is totally a hack...  I'll give you some points for it, but the correct fix is to add a "hint" or "region" column to the data files and use that.  That'd be worth a lot more points, obviously.